### PR TITLE
Add new project "EventsOfWeek"

### PR DESCRIPTION
**Pull request by arturyaroshinski**
This project implements a primitive version of the app for recording events on a day of the week.

### CODE REVIEW

File: src/EventsOfWeek/Program.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
+﻿using System;
+
+namespace EventsOfWeek
+{
+    class Program
+    {
+        enum Week
+        {
+            Sunday = 0,
```
Comment (MikhailMasny) [inline src/EventsOfWeek/Program.cs]: Maybe better when sunday at number 7? 🔢

**Comment by arturyaroshinski** [comment]
Changed them

File: src/EventsOfWeek/Program.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        static void SelectDay()
+        {
+            string userInput = Console.ReadLine();
+            switch (userInput.ToLower())
+            {
+                case "mon":
+                case "monday":
+                case "1":
+                    Console.WriteLine("U choose monday.");
+                    AddEvent(Convert.ToInt32(Week.Monday));
```
Comment (MikhailMasny) [inline src/EventsOfWeek/Program.cs]: Try to send to method enum param. It is look like: AddEvent((int)Week.Monday);
Or try to work with enum at AddEvent method (this case better).

**Comment by arturyaroshinski** [comment]
Reworked the method by choosing 2 case(if I understood it correctly).

**Review by MikhailMasny — APPROVED** [review]
Exemplary PR! 🥇

**Pull request merged**